### PR TITLE
update reservation dependency file

### PR DESCRIPTION
**Pull request by Only2125**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by haroldrandom** [comment]
Why update the setup.py? Any context?

**Comment by Only2125** [comment]
> A hint, the dependency not just in setpy.py , but also requirements.py2.Darwin.txt, requirements.py3.Darwin.txt, requirements.py2.windows.txt, requirements.py3.windows.txt, requirements.py2.Linux.txt, requirements.py3.Linux.txt
> Besides, Why have to update the SKD?

Hi, thanks for the reply. We have an update for sdk, release some new features. So we want to update the CLI. But I may made some mistake here. I will try to fix it.

**Comment by haroldrandom** [comment]
**HAVE all the TESTS ran well on your local machine?** 
According to CI report, your module about `azure-mgmt-reservations` is not found, please make sure of that.

**Comment by haroldrandom** [comment]
kindly remind, these changes will influence user, so HISTORY.rst must be updated too.

File: src/azure-cli/azure/cli/command_modules/reservations/tests/latest/test_reservations_commands.py [hunk 1]
```diff
@@ -3,10 +3,10 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from azure.cli.testsdk import ScenarioTest, record_only
+from azure.cli.testsdk import ScenarioTest
+
 
 
-@record_only()
 class AzureReservationsTests(ScenarioTest):
```
Comment (haroldrandom) [inline src/azure-cli/azure/cli/command_modules/reservations/tests/latest/test_reservations_commands.py]: Too many blank lines, please check CI

Comment (haroldrandom) [inline src/azure-cli/azure/cli/command_modules/reservations/tests/latest/test_reservations_commands.py]: Too many blank lines, please check CI

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+**Reservations**
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst]: Inline emphasis start-string without end-string, please check CI

**Comment by haroldrandom** [comment]
Any Update?

**Comment by Only2125** [comment]
> Any Update?

Hi, I found we have an issue in Swagger. swagger change merged, working on sdk.

**Comment by Only2125** [comment]
Create a brand new PR. Abandon this one

**Comment by haroldrandom** [comment]
Reopen here #11126

**Pull request closed without merging**